dependencies/maven/dependencies.yaml

```diff
@@ -56,4 +56,14 @@ dependencies:
       lang: java
     netty-tcnative-boringssl-static:
       version: "2.0.25.Final"
+      lang: java
+
+  org.slf4j:
+    slf4j-api:
+      version: "1.7.28"
+      lang: java
```

Comment:
Is this the default logger we use across @graknlabs @adammitchelldev @lolski ?

Reply:
Yes, `org.slf4j.slf4j-api` is our standard logger.

Reply:
👍